src/Sql/Sql.Test/ScenarioTests/RecommendedActionTests.cs

```diff
@@ -27,21 +27,21 @@ public RecommendedActionTests(ITestOutputHelper output) : base(output)
 
         #region Server Recommended Action Tests
 
-        [Fact]
+        [Fact(Skip = "Resource group 'WIRunnersProd' could not be found.")]
```

Comment:
Could you please re-create the resource group and the resources needed to run this test?

Reply:
Done